.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
       - run:
           name: install bundler
-          command: gem install bundler
+          command: gem install bundler -v '~> 2.3.27'
```

Comment:
It was trying to install too new of a bundler

Reply:
If we're going to upgrade this gem to ruby 3.1.4, then we should be using bundler ~> 2.5.0 to match the other apps.